.changeset/chilly-pets-walk.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Would this be a breaking change? As it seems the original type infer does not work any more

Reply:
this was an experimental option and it doesn't have any effect now